src/sidebar/components/test/group-list-item-test.js

```diff
@@ -36,64 +34,54 @@ describe('groupListItem', () => {
       track: sinon.stub(),
       events,
     };
-
-    angular.mock.module('app', {
-      analytics: fakeAnalytics,
-      store: fakeStore,
-    });
   });
 
   const createGroupListItem = fakeGroup => {
-    return util.createDirective(document, 'groupListItem', {
-      group: fakeGroup,
-    });
+    return mount(
```

Comment:
Yes, there are three rendering options. Full ("mount" 🤷‍♂️), shallow and string (produces a string of HTML instead of DOM nodes). In this case it doesn't make much difference since there are no children, shallow would work equally well. "Shallow" is I think what we'll probably want to use as a default.